javascript/src/main/utils/fetch.js

```diff
@@ -10,6 +10,7 @@ export async function fetchWithToken(url, getToken, options) {
     },
   });
   if (response.status >= 400 && response.status < 600) {
+    console.log(response.error_description)
```

Comment:
Logging errors in production can be a potential security leak (depending on what information appears in the error). The most I would do is `console.err("There was an error communicating to the backend")`, especially when this function is used for _multiple_ api calls.